modules/backend/layouts/default.htm

```diff
@@ -44,15 +44,15 @@
                                 <div class="layout">
                                     <?php if ($breadcrumbContent = Block::placeholder('breadcrumb')): ?>
                                         <!-- Breadcrumb -->
-                                        <div class="control-breadcrumb">
+                                        <div class="control-breadcrumb" aria-label="Breadcrumb Navigation">
                                             <?= $breadcrumbContent ?>
                                         </div>
                                     <?php endif ?>
 
                                     <!-- Content -->
-                                    <div class="layout-row">
+                                    <div class="layout-row" aria-label="Main Content Body">
                                         <?= Block::placeholder('body') ?>
-                                    </div>
+                                    </div>                                     
```

Comment:
lot of extra whitespace here that's unnecessary

Reply:
fixed.